week2/homework/src/index.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
 // TODO: Write the homework code in this file
+'use strict';
+// const program = require('commander');
+
+const command = process.argv[2];
+const argument = process.argv.slice(3);
+const renderList = require('./todolist');
+const addToList = require('./addtodo');
+const removeToDo = require('./removetodo');
+const reset = require('./reset');
+const update = require('./update');
+const helpCommand = require('./help');
+function actions(commandsCallFunc) {
+  commandsCallFunc.then(function() {
+    return renderList();
+  });
+}
```

Comment:
Neat!
But also don't forget to handle the errors, so `.catch(// ...`

Reply:
thank you :)